### ISSUE TITLE
Dialog with items bug.

### ISSUE THREAD

**Issue by fngr** [danoz73/QustomDialog#2]
Hi,

I have been testing your dialog and it's a nice work but i have found an issue.

When you use the method .setItems, to build a dialog with a list of items to choose, the list appear before the header.

Cheers.

**Comment by danoz73** [comment]
Yeah, this was easily reproducible. I'll try to fix this issue soon.

**Comment by inssein** [comment]
@danoz73 any updates? I don't mind trying to fix it if you could guide me a little bit (I'm very new to android)
